examples/python/openai-story-writer/client.py

```diff
@@ -0,0 +1,15 @@
+from acp_sdk.client import Client
+from acp_sdk.models import Message, MessagePart
+
+import asyncio
+
+async def run_agent(agent: str, input: str) -> list[Message]:
+    async with Client(base_url="http://localhost:8000") as client:
+        run = await client.run_sync(
+            agent=agent, input=[Message(parts=[MessagePart(content=input, content_type="text/plain")])]
```

Comment:
This should be possible now:

```suggestion
            agent=agent, input=input
```